Modules/ChequeModule/Views/ChequeView.xaml

```diff
@@ -17,17 +18,160 @@
         </ResourceDictionary>
     </UserControl.Resources>
 
-    <Grid Background="{StaticResource Secondary}">
+    <Grid Background="{StaticResource Secondary}" >
+
         <Grid.RowDefinitions>
             <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
+            <RowDefinition Height="17*" />
```

Comment:
`*` will takes up remaining space after fixed sizes are allocated
`n*` allocates a proportion of the available space based on the weight specified by the `n`.

In Full screen:
![image](https://github.com/kevinboon3288/cheque-writer/assets/38966644/0595fcc8-1fe5-4231-bbea-4b84b0562685)

In small resolution:
![image](https://github.com/kevinboon3288/cheque-writer/assets/38966644/41fd3b65-d80a-44b5-afbc-468a099bc522)

It will be harder to read and not efficient to construct the layout of the component or view.
May try to use `auto` and only `*`.